### PR TITLE
dev-python/ijson: bump v2.5.1

### PR DESCRIPTION
**Pull request by arthurzam**
Maintenance over this package was passed to new developer, based on
previous maintainer [comment](https://github.com/isagalaev/ijson/pull/58#issuecomment-500596815). Updated the links in new version and `metadata.xml`

- Moved `dev-python/setuptools` to `BDEPEND`
- The comment around `python_test()` is still relevant
- Just to be sure, recheck the license (it is `BSD`)

Package-Manager: Portage-2.3.78, Repoman-2.3.17

### CODE REVIEW

**Comment by arthurzam** [comment]
Bump.

File: dev-python/ijson/ijson-2.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-libs/yajl"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+# pypi release lacks tests and git release isn't tagged yet
```
Comment (mgorny) [inline dev-python/ijson/ijson-2.5.1.ebuild]: It's tagged now.

Comment (arthurzam) [reply dev-python/ijson/ijson-2.5.1.ebuild]: Well I agree that it is tagged, but the tests are excluded from the `pypi` release - checked now.
Maybe update the comment, but I'm not sure to what.
I did manually run on the `github` tag release, so maybe switch to it?

Comment (mgorny) [reply dev-python/ijson/ijson-2.5.1.ebuild]: Yes, just use github snapshot.

Comment (arthurzam) [reply dev-python/ijson/ijson-2.5.1.ebuild]: Updated to use `github` release, just to be sure ran `ebuild ... test` for `python2_7` and `python3_7` - Successful for all 57 tests.

**Comment by arthurzam** [comment]
Thank you very much.
By the way, I have many more `dev-python/*` simple version bumps. I think using Pull Requests for every one of them is not efficient. I'm not against becoming a Gentoo developer just for simple Python bumps (I'm afraid I still have to study for a full blown maintainer). What do you suggest?

**Comment by mgorny** [comment]
If you are sure no changes are required (including no missing RESTRICT) and tests pass, you can assemble me a list and e.g. paste it on some bug or send via mail.

**Pull request closed without merging**